engine/src/main/java/org/openobservatory/engine/Engine.java

```diff
@@ -42,15 +28,18 @@ public static OONISession newSession(OONISessionConfig config) throws Exception
     public static OONISessionConfig getDefaultSessionConfig(Context ctx,
```

Comment:
I honestly don't like this kind of factory/constructor. I think it would be more explicit to initialize fields in place.